internal/lib/stop.go

```diff
@@ -2,7 +2,6 @@ package lib
 
 import (
 	"context"
-
```

Comment:
we typically separate the stdlib imports from the ones imported by go modules

Reply:
fixed